docs/package_templates/cmake_package/all/conanfile.py

```diff
@@ -58,12 +57,9 @@ def configure(self):
                 del self.options.fPIC # once removed by config_options, need try..except for a second del
             except Exception:
                 pass
-        try:
-            del self.settings.compiler.libcxx # for plain C projects only
-        except Exception:
-            pass
         try:
             del self.settings.compiler.cppstd # for plain C projects only
+            del self.settings.compiler.libcxx # for plain C projects only
```

Comment:
Revert. We can't guarantee cppstd is evaluated.

Reply:
I gotcha, hadn't thought of that. Done.